src/Bonsai.Zyre/ToZyreShout.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Reactive.Linq;
+using NetMQ;
+
+namespace Bonsai.Zyre
+{
+    /// <summary>
+    /// Represents an operator that packages a <see cref="NetMQMessage"/> as a Zyre shout message.
+    /// </summary>
+    public class ToZyreShout : Transform<Tuple<NetMQMessage, string>, ZyreMessage>
+    {
+        /// <summary>
+        /// Transforms and observable sequence of messages and target groups to an observable sequence of Zyre shout messages.
+        /// </summary>
+        /// <param name="source">
+        /// A union of <see cref="NetMQMessage"/> containing the message data and a <see cref="string"/> referencing the target group of the shout.
+        /// </param>
+        /// <returns>
+        /// An observable sequence of <see cref="ZyreMessageShout"/>.
+        /// </returns>
+        public override IObservable<ZyreMessage> Process(IObservable<Tuple<NetMQMessage, string>> source)
+        {
+            return source.Select(x => new ZyreMessageShout { CommandType = ZyreCommandType.Shout, Group = x.Item2, Message = x.Item1 });
```

Comment:
Having `Group` / `Peer` specified as part of the input tuple is hard to understand since tuples are nameless relative to their parameters and so harder to discover in the IDE. I was thinking of two alternatives:
  1. Specify `Group` / `Peer` as a public property. This would mirror the `Topic` property in the publisher socket operator of the ZMQ package.
  2. Follow the router pattern and allow for the group / peer name to be specified in the first frame of the message.

These don't necessarily have to be mutually exclusive, but maybe 1. would be simpler to implement for now. I might merge this as it is, but would be curious to hear your thoughts.